### PR TITLE
[D&D] Fixes time series for new chart types

### LINKED ISSUE #2307: [D&D] [BUG] Timeline fields in Wizard do not work
**Describe the bug**

Histograms with time ranges in the X axis do not work

**To Reproduce**
1. Go to wizard
2. Drag count to the Y axis and click add. anew field in the X Axis
3. Select 'Date Histogram' in the fields section

**Expected behavior**
We should be able to see a date histogram.

**OpenSearch Version**
2.3

**Dashboards Version**
2.3

**Plugins**

Please list all plugins currently enabled.

**Screenshots**

**Host/Environment (please complete the following information):**
 - Browser and version [e.g. 22]

**Additional context**

### PR DESCRIPTION
**Pull request by ashwin-pc**
Signed-off-by: Ashwin Pc <ashwinpc@amazon.com>

### Description
Fixes rendering timeseries for the bar, line and area charts.

image
 
### Issues Resolved
#2307 
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
    - [ ] `yarn test:jest`
    - [ ] `yarn test:jest_integration`
    - [ ] `yarn test:ftr`
- [ ] New functionality has been documented.
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/wizard/public/visualizations/vislib/common/get_pipeline_params.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { BuildPipelineParams } from '../../../../../visualizations/public';
+import { getTimeFilter } from '../../../plugin_services';
+
+export const getPipelineParams = (): BuildPipelineParams => {
+  const timeFilter = getTimeFilter();
+  return {
+    // TODO: this will blow up for time x dimensions
```
Comment (joshuarrrr) [inline src/plugins/wizard/public/visualizations/vislib/common/get_pipeline_params.ts:12]: Isn't this what the PR fixes?

File: src/plugins/wizard/public/visualizations/vislib/area/to_expression.ts [hunk 2]
```diff
@@ -7,24 +7,20 @@ import { Vis, buildVislibDimensions } from '../../../../../visualizations/public
 import { buildExpression, buildExpressionFunction } from '../../../../../expressions/public';
 import { AreaOptionsDefaults } from './area_vis_type';
 import { getAggExpressionFunctions } from '../../common/expression_helpers';
-import { VislibRootState } from '../common/types';
-import { getValueAxes } from '../common/get_value_axes';
+import { VislibRootState, getValueAxes, getPipelineParams } from '../common';
```
Comment (joshuarrrr) [inline src/plugins/wizard/public/visualizations/vislib/area/to_expression.ts:10]: nice

**Review by joshuarrrr — COMMENTED** [review]
Is the TODO comment still valid? Otherwise, looks good!

**Comment by ashwin-pc** [comment]
Removed the TODO :)

**Review by kavilla — APPROVED** [review]

**Pull request merged**